README.md

```diff
@@ -218,9 +208,13 @@ curl localhost:8080/metrics | grep ^notification_writer
 
 ## Database
 
-PostgreSQL database is used as a storage.
+PostgreSQL database is used as a storage for new reports and for already reported reports as well.
```

Comment:
for both new and already notified reports.

Reply:
yup, will change it